app/models/criterion.rb

```diff
@@ -165,4 +165,17 @@ def visible?
       false
     end
   end
+
+  public
```

Comment:
Generally you shouldn't use this keyword. You just need to move this method to before the `private` keyword above.